Models/DeliveryModel.go

```diff
@@ -0,0 +1,39 @@
+package Models
+
+import (
+	"errors"
+	"fmt"
+	"github.com/Baraulia/COURIER_SERVICE/db"
+	"log"
+)
+
+type DeliveryService struct {
+	repo db.Repository
+}
+
+func NewDeliveryService(repo db.Repository) *DeliveryService {
+	return &DeliveryService{repo: repo}
+}
+
+func (s *DeliveryService) GetOrders() ([]db.Order, error) {
+	var Orders []db.Order
+	err := s.repo.GetActiveOrdersFromDB(&Orders)
+	if err != nil {
+		return nil, fmt.Errorf("%w", err)
```

Comment:
When you return wrapped error you need to provide some extra context around it, otherwise it is unclear why to wrap it if you can just return initial error